internal/api/consts.go

```diff
@@ -0,0 +1,6 @@
+package api
+
+const (
+	NodeIDHashLabelKey   = "csi.cert-manager.io/node-id-hash"
+	VolumeIDHashLabelKey = "csi.cert-manager.io/volume-id-hash"
```

Comment:
VolumeID is expected to base unique *within* a node - so we have both the volume ID and the node ID as part of the 'key' when looking up what is a candidate to delete